events/wc_void_events.txt

```diff
@@ -915,152 +915,150 @@ narrative_event = {
 
 ### The Reawakening of C'thun ###
 # Ping event
-province_event = {
-	id = WCVOD.1000
-
-	is_triggered_only = yes
-	hide_window = yes
+# province_event = {
+	# id = WCVOD.1000
```

Comment:
Now C'thun don't awake in 608

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/68530156-f56dd080-0316-11ea-9695-c9e5e2d41eb6.png)
![image](https://user-images.githubusercontent.com/46576860/68530167-00c0fc00-0317-11ea-9116-fbb41a4779c0.png)
![image](https://user-images.githubusercontent.com/46576860/68530171-146c6280-0317-11ea-907c-d11f397a748f.png)


</details>